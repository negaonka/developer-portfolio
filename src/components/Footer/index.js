import React from 'react';
import github from '../../assets/social/github.png';
import linkdln from '../../assets/social/linkdln.jfif';
import stackOverflow from '../../assets/social/stackOverflow.jfif';

function Footer(){

    const footerImages = [{name: github}, {name: linkdln}, {name: stackOverflow}]
    return(
        <footer>
            <div className="flex-row" style={{justifyContent: "center"}}>

            {footerImages.map((footerImage) => (
             <img src={footerImage.name} className="px-1 py-1 img-thumbnail mx-1"
             style={{ width: "60px", height: "60px"}} alt="cover" />
            ))}
            </div>
        </footer>
    )
}

export default Footer;