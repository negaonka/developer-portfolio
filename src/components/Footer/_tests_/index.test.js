import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup);  //to clean all the memory data

describe('Footer component', () => {
    // baseline test
    it('renders', () => {
        render(<Footer/>);
      });
   
    // snapshot test
    it('matches snapshot', () => {
        const { asFragment } = render(<Footer/>);
        // assert value comparison
        expect(asFragment()).toMatchSnapshot();
      });

    })