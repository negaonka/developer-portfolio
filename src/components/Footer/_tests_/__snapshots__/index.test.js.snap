// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="flex-row"
      style="justify-content: center;"
    >
      <img
        alt="cover"
        class="px-1 py-1 img-thumbnail mx-1"
        src="github.png"
        style="width: 60px; height: 60px;"
      />
      <img
        alt="cover"
        class="px-1 py-1 img-thumbnail mx-1"
        src="linkdln.jfif"
        style="width: 60px; height: 60px;"
      />
      <img
        alt="cover"
        class="px-1 py-1 img-thumbnail mx-1"
        src="stackOverflow.jfif"
        style="width: 60px; height: 60px;"
      />
    </div>
  </footer>
</DocumentFragment>
`;
